table/sst_file_writer.cc

```diff
@@ -7,6 +7,7 @@
 
 #include <vector>
 
+#include "db/db_impl/db_impl.h"
```

Comment:
Why is this necessary?

Reply:
Not needed anymore, since we don't need to sanitize options here anymore.